recipes/libnl-cos6-x86_64/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: libnl-cos6-x86_64
+  version: 1.1.4
+
+source:
+  - url: http://mirror.centos.org/centos/6.10/os/x86_64/Packages/libnl-1.1.4-2.el6.x86_64.rpm
+    sha256: 03a7235e747695208a2633fd6515a39377a9b06b7aa866bda1a1e1eabf418869
+    folder: binary
+  - url: http://vault.centos.org/6.10/os/Source/SPackages/libnl-1.1.4-2.el6.src.rpm
```

Comment:
@jakirkham, can you add a sha256 for this?

Reply:
Please see issue ( https://github.com/conda/conda-build/issues/3552 ).

Reply:
That doesn't mean, it couldn't be added manually. See https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L19 for instructions

Reply:
I dropped this from the feedstock. It is unused AFAICT. ( https://github.com/conda-forge/libnl-cos6-x86_64-feedstock/pull/1 )